pages/pipelines/emojis.md.erb

```diff
@@ -38,3 +38,8 @@ A few common emojis are listed below, but you can see the [full list of availabl
 ## Adding Custom Emojis
 
 Add your own emoji by opening a [pull request](https://github.com/buildkite/emojis#contributing-new-emoji) containing a 64x64 PNG image and a name to the emoji repository.
+
+<section class="Docs__troubleshooting-note">
+  <h3>Your SCM may not display custom emojis correctly</h3>
+  <p>Note that custom emojis that are supported in many Buildkite fields may not render on GitHub or other SCM services.</p>
```

Comment:
```suggestion
  <h3>Buildkite emojis in other tools</h3>
  <p>Buildkite loads custom emojis as <a href="https://github.com/buildkite/emojis">images</a>. Other tools might not display the images correctly, and will only show the <code>:text-form:</code>.</p>
```

Reply:
I'm also lobbying pretty hard on not using SCM.

Reply:
Totally fine with that. I learned the term SCM at Buildkite. I hadn't really heard it before. Let's go with what you said.